rewards/snapshot/token_snapshot.py

```diff
@@ -1,14 +1,20 @@
-from typing import Dict, Tuple
+from typing import Counter, Dict, Tuple
 
 from helpers.constants import BADGER, DIGG
 from rewards.classes.Snapshot import Snapshot
-from subgraph.queries.tokens import fetch_fuse_pool_balances, fetch_token_balances
+from subgraph.queries.tokens import (
+    fetch_across_balances,
+    fetch_fuse_pool_balances,
+    fetch_token_balances,
+)
 from subgraph.subgraph_utils import make_gql_client
 
 
 def token_snapshot(chain: str, block: int) -> Tuple[Snapshot, Snapshot]:
     token_client = make_gql_client(f"tokens-{chain}")
     badger_bals, digg_bals = fetch_token_balances(token_client, block, chain)
+    across_bals = fetch_across_balances(block, chain)
+    badger_bals = Counter(badger_bals) + Counter(across_bals)
```

Comment:
Small thing: what do you think if we don't override `badger_bals` but create some new variable to avoid mutations?
Like
```python
cummulative_balances = Counter(badger_bals) + Counter(across_bals)
return Snapshot(BADGER, cummulative_balances), Snapshot(DIGG, digg_bals)
```